lib/logstash/plugin_mixins/jdbc.rb

```diff
@@ -72,6 +72,9 @@ def setup_jdbc_config
     # examples of vendor-specific options can be found in this
     # documentation page: https://github.com/jeremyevans/sequel/blob/master/doc/opening_databases.rdoc
     config :sequel_opts, :validate => :hash, :default => {}
+
+    # Log level at which to log SQL queries. The default value is info.
+    config :sql_log_level, :validate => :string, :default => "info"
```

Comment:
can we restrict this to accepted values? Also can you add this to the doc right above this line?